mode/dockerfile/dockerfile.js

```diff
@@ -26,22 +26,62 @@
         regex: /#.*$/,
         token: "comment"
       },
+      {
+        regex: /^(\s*)\b(?:(from))\b/i,
```

Comment:
You can just drop the non-capturing group.

Reply:
Done.